src/scripts/main.js

```diff
@@ -1 +1,82 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject(new Error());
+  }, 3000);
+});
+
+const secondPromise = new Promise(resolve => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+
+  document.addEventListener('contextmenu', (ev) => {
+    ev.preventDefault();
+    resolve();
+  });
+});
+
+const thirdPromise = new Promise(resolve => {
+  let rightClick = false;
+  let leftClick = false;
+
+  document.addEventListener('mouseup', (ev) => {
+    if (ev.button === 0) {
+      leftClick = true;
+    }
+
+    if (ev.button === 2) {
+      rightClick = true;
+    }
+
+    if (leftClick && rightClick) {
+      resolve();
+    }
+  });
+});
+
+firstPromise
+  .then(() => {
+    showMessage('first', 'success');
+  })
+  .catch(() => {
+    showMessage('first', 'error');
+  });
+
+secondPromise
+  .then(() => {
+    showMessage('second', 'success');
+  });
```

Comment:
You do the same action on success for each promise. Create a method for it, please